modules/configuring-a-pipeline-server-with-an-external-amazon-rds-db.adoc

```diff
@@ -0,0 +1,57 @@
+:_module-type: PROCEDURE
+
+[id='configuring-a-pipeline-server-with-an-external-amazon-rds-db_{context}']
+= Configuring a pipeline server with an external Amazon RDS database
+
+To configure a pipeline server with an external Amazon Relational Database Service (RDS) database, you must configure {productname-short} to trust the certificates issued by its certificate authorities (CA).
+
+[IMPORTANT]
+====
+If you are configuring a pipeline server for production pipeline workloads, {org-name} recommends that you use an external MySQL or MariaDB database. 
+====
+
+.Prerequisites
+* You have cluster administrator privileges for your {openshift-platform} cluster.
+* You have logged in to {productname-long}.
+* You have created a data science project that you can add a pipeline server to.
+* You have an existing S3-compatible object storage bucket, and you have configured your storage account with write access to your S3 bucket.
+
+.Procedure
+. Before configuring your pipeline server, from link:https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.SSL.html#UsingWithRDS.SSL.CertificatesAllRegions[Amazon RDS: Certificate bundles by AWS Region], download the PEM certificate bundle for the region that the database was created in. 
++
+For example, if the database was created in the `us-east-1` region, download `us-east-1-bundle.pem`.
+. In a terminal window, log in to the OpenShift cluster where {productname-short} is deployed.
```

Comment:
In the module modules/accessing-s3-compatible-object-storage-with-self-signed-certificates.adoc, most of the sentences containing commands are ended with a colon (apart from the one that I highlighted). Here, in this module, all such sentences are ended with a full stop.